addons/im_livechat/static/src/core/common/thread_actions_patch.js

```diff
@@ -0,0 +1,43 @@
+import { threadActionsRegistry } from "@mail/core/common/thread_actions";
+import { patch } from "@web/core/utils/patch";
+
+patch(threadActionsRegistry.get("invite-people"), {
+    condition(component) {
+        return (
+            component.thread?.model === "discuss.channel" &&
+            (!component.props.chatWindow || component.props.chatWindow.isOpen) &&
+            !component.thread?.composerDisabled
```

Comment:
Shouldn't these conditions be on `livechat_active` instead? `composerDisabled` looks too specific to another purpose.

Also, it would be nice to call super && ... rather than redefining everything.
+ don't we want to apply the override code only for livechat channel type?